redisless/src/server.rs

```diff
@@ -346,6 +368,11 @@ mod tests {
         let x: String = con.get("key2").unwrap();
         assert_eq!(x, "value2");
 
+        let _: () = con.set("intkey", "10").unwrap();
+        con.send_packed_command(cmd("INCR").arg("intkey").get_packed_command().as_slice()).unwrap();
```

Comment:
From what I see in the Redis client documentation, there is a `con.incr(K)` method. https://docs.rs/redis/0.20.0/redis/trait.Commands.html#method.incr

Reply:
The `con.incr(K, V)` method issues an `INCRBY` or an `INCRBYFLOAT` command. There is no `con.incr(K)` method.